libmisc/find_new_gid.c

```diff
@@ -112,6 +113,11 @@ static int check_gid (const gid_t gid,
 		return ERANGE;
 	}
 
+  /* Check for compatibility with 16b and 32b gid_t error codes */
```

Comment:
Indentation issue

Reply:
Sorry, I did not set up everything properly looks like. Is the preferred default the same as with linux (indentation with tabs of width 8)?

Reply:
I don't think we have any preference, we just try to keep the same indentation throughout the file.